Content.Client/UserInterface/HandsGui.cs

```diff
@@ -239,10 +254,10 @@ protected override void KeyBindDown(GUIBoundKeyEventArgs args)
         {
             base.KeyBindDown(args);
 
-            if (!args.CanFocus)
-            {
-                return;
-            }
+            //if (!args.CanFocus)
+            //{
+            //    return;
+            //}
```

Comment:
Getting rid of this was necessary for ActivateItemInWorld to work, since it appeared to have `CanFocus = false`. I don't really understand the why and how of this, so insight into if this check is actually important would be appreciated.

Reply:
Removing it is fine. That said, just remove it entirely instead of commenting it out.